server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -366,6 +367,107 @@ public void subscriptionUserShouldBeChangedWhenUpdateMailbox() throws Exception
             .doesNotContain(initialMailboxName);
     }
 
+    @Test
+    public void subscriptionUserShouldBeChangedWhenUpdateParentMailbox() throws Exception {
+        MailboxId parentId = mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, username.asString(), "root");
+
+        String secondMailboxName = "second";
+        createSubMailBox(secondMailboxName, parentId.serialize());
+        String secondMailboxId = mailboxProbe.getMailboxId(MailboxConstants.USER_NAMESPACE, username.asString(), "root.second").serialize();
```

Comment:
Can we make createSubMailBox return a mailboxId so that we don't need to get it in a second step?